ecosystem/sep-0045.md

```diff
@@ -0,0 +1,431 @@
+**Server**. A **Server** is required to implement the following behavior for the web authentication endpoint:
+
+- [`GET <WEB_AUTH_ENDPOINT_SEP0045>`](#challenge): request a challenge (step 1)
+- [`POST <WEB_AUTH_ENDPOINT_SEP0045>`](#token): exchange a signed challenge for session JWT (step 2)
+
+## Cross-Origin Headers
+
+Valid CORS headers are necessary to allow web clients from other sites to use the endpoints. The following HTTP header
+must be set for all authentication endpoints, including error responses.
+
+```
+Access-Control-Allow-Origin: *
+```
+
+In order for browsers-based wallets to validate the CORS headers, as
+[specified by W3C](https://www.w3.org/TR/cors/#preflight-request), the preflight request (OPTIONS request) must be
+implemented in all the endpoints that support Cross-Origin.
+
+### Challenge
+
+This endpoint must respond with authorization entries to be signed by the **Client**. The **Client** signs the entries
+using standard Stellar libraries and submit it to [`token`](#token) endpoint to prove that it controls the **Client
+Account**. This approach is compatible with hardware wallets such as Ledger. The **Client Application** must also verify
+the server's signature to be sure the challenge is signed by the **Home Domain Address**, that the home domain argument
+in the function invocation is the **Home Domain**, and that the web auth domain argument in the function invocation is
+the **Server** domain.
+
+#### Request
+
+```
+GET <WEB_AUTH_ENDPOINT_SEP0045>
+```
+
+##### Request Parameters:
+
+| Name            | Type          | Description                                                                                                                                                                                                                                                                                                                                                                                                                |
+| --------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `address`       | `C...` string | The **Client Account** address (`C...`) that the **Client** wishes to authenticate with the **Server**.                                                                                                                                                                                                                                                                                                                    |
```

Comment:
I can understand why renaming this field from account (SEP-10) to address (SEP-45), but it also makes the SEP-45 and SEP-10 APIs use different terms for the same thing. I don't think we benefit much from renaming concepts in SEP-45, it could use the same vocab as SEP-10 unless necessary.